articles/azure-app-configuration/howto-import-export-data.md

```diff
@@ -479,6 +479,7 @@ From the Azure CLI, follow the steps below. If you don't have the Azure CLI inst
     | `--destination`        | Enter `appservice` to indicate that you're exporting data to App Service.                                                                                          | `appservice`                                                                                                              |
     | `--appservice-account` | Enter the App Service's ARM ID or use the name of the App Service, if it's in the same subscription and resource group as the App Configuration.                   | `/subscriptions/123/resourceGroups/my-as-resource-group/providers/Microsoft.Web/sites/my-app-service` or `my-app-service` |
     | `--label`              | Enter a label to export keys and feature flags with this label. If you don't specify a label, by default, you will only export keys and feature flags with no label. You can enter one label, enter several labels by separating them with `,`, or use `*` to take all of the labels in account. | `prod`                                                                                                                    |
+    | `--export-as-reference` `-r` | Optional. Flag to specify whether key-values are exported to App Service as App Configuration references. This exports references to the latest revision of key-value(s) in App Configuration. [Learn more](../app-service/app-service-configuration-references.md). |
 
```

Comment:
I can't add a comment on line 507. Can you update the link of "az appconfig kv export" to "/cli/azure/appconfig/kv#az-appconfig-kv-export"? The existing one doesn't work for me.

Reply:
This link refers to an autogenerated page from the [_help.py](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/appconfig/_help.py) page in our cli repo, that is why it does not work when directly clicked within the  vs code preview, because such a file does not exist in the docs repo. But this works as expected when clicked in the deployed version [here](https://learn.microsoft.com/en-us/azure/azure-app-configuration/howto-import-export-data?tabs=azure-cli#export-data-to-azure-app-service:~:text=Export%20data%20to%20Azure%20App%20Service).

Reply:
I didn't click from with VS code. The link doesn't jump to the right section of the doc. Something is wrong with the `preserve-view=true` portion of the link. Can you please update the link as I suggested earlier?

Reply:
I see it now. I missed the suggestion. My apologies for that.